Podfile.lock

```diff
@@ -79,4 +79,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 9d2173089b033c8fcf43f785a45f318702e5da67
 
-COCOAPODS: 1.3.1
+COCOAPODS: 1.2.1
```

Comment:
Please update the cocoapods version locally on your system.

Reply:
I want to know if when i make my next PR, it'll be necessary to go through the automated test. Because, the log in isn't working, and there's a test for log in that always fails, avoiding the automatic merging. Also, i'm having issues with the xcode 9 beta, so i am installing the one from the app store... it will take a little while.

Reply:
It's better if you pass the tests. Regarding the login, changing http to https somehow fixes that problem. Haven't tried running the app on xcode 9 yet so can't say.

Reply:
Okay, i've got xcode installed back now it's time to try it.

Reply:
Please update cocoapods locally since this does not need to be changed.

Reply:
I have done that now and i have pushed